jdaviz/core/template_mixin.py

```diff
@@ -1,26 +1,34 @@
-import ipyvuetify as v
+from warnings import warn
+
+from ipyvuetify import VuetifyTemplate
 from glue.core import HubListener
-from traitlets import List
 
 __all__ = ['TemplateMixin']
 
 
-class TemplateMixin(v.VuetifyTemplate, HubListener):
+class TemplateMixin(VuetifyTemplate, HubListener):
     def __new__(cls, *args, **kwargs):
         """
         Overload object creation so that we can inject a reference to the
         ``Hub`` class before components can be initialized. This makes it so
         hub references on plugins can be passed along to components in the
         call to the initialization method.
         """
-        app = kwargs.pop('app', None)
         obj = super().__new__(cls, *args, **kwargs)
-        setattr(obj, '_app', app)
+        obj._app = kwargs.pop('app', None)
```

Comment:
We want to pop `app` before the `kwargs` dict is passed to `super` -- `app` can cause issues when passed to parent classes.

Reply:
Ah, gotcha. I meant this really to just change the setattr but that totally makes sense, my bad, will fix.